### DISCUSSION TITLE
Removing Saturdays and Sundays from "Weekly working hours"

### DISCUSSION THREAD

**Discussion by afcec** [kimai/kimai#3071] [Questions & Answers] [answered]
Hi, first of all I would like to thank you for this great application. I just got it working last week and so far everything is perfect. 

I have one question regarding the "Weekly working hours", which is the option my colleagues use more frequently: is there any way to remove/hide Saturdays and Sundays? There is a lot of information in that web page and when the project and/or activity names are long, let´s say the overall view gets quite condensed, compressed. We do not work during weekends and if we do, we do not assign effort to those "non-official" days so it would be great to remove them. However, I have not been able to find any configuration option allowing me to define the active working days for the users.

Thanks for your help

**Comment by kevinpapst** [comment, marked as answer]
This is not yet possible. The size of the project and activity field need to be limited (open TODO) and hiding the weekends could re-use the calendar settings. I'll think about it!

**Comment by SaturnusDJ** [comment]
This topic has been awhile but just did this.
In case it is only about the aesthetics...custom CSS:
```
#qe-totals-day-5, #qe-totals-day-6, table.dataTable td.weekend, table.dataTable th.weekend {
  display: none;
}
```
(The duration column will be 'wrong' if in another way entries are entered for Sat/Sun.)
